garamohamed98/storesight-ai
# Add Topbar with notification icon and dark/light mode toggle function…

# What was done:

- Added a Topbar component to the header of the application.
- Included a notification icon, which is currently non-functional.
- Implemented a dark/light mode toggle button to switch between themes.

# Why it was done:

- Enhances the user interface by providing a consistent navigation bar at the top of the page.
- Prepares for future functionality with the notification icon.
- Allows users to switch between light and dark modes for better usability in varying lighting conditions.

# How to test:

- Check the Topbar at the top of the page for the new component.
- Use the dark/light mode toggle button to switch between themes.
- Confirm that the notification icon is visible but does not perform any actions.

# Screenshots/GIFs:

![image](https://github.com/user-attachments/assets/a386d5e7-d0e6-476d-acb7-c8a209454a75)
